src/tests/ftest/util/job_manager_utils.py

```diff
@@ -296,11 +296,9 @@ def __init__(self, job, subprocess=False, mpitype="openmpi"):
         self.processes = FormattedParameter("-np {}", 1)
         self.ppn = FormattedParameter("-ppn {}", None)
         self.envlist = FormattedParameter("-envlist {}", None)
-        self.mca = FormattedParameter("--mca {}", mca_default)
-        self.working_dir = FormattedParameter("-wdir {}", None)
-
         self.mpitype = mpitype
         self.working_dir = FormattedParameter("-wdir {}", None)
+        self.mca = FormattedParameter("--mca {}", None)
```

Comment:
This change will cause issues with other tests.  Please revert back to the original file.

Reply:
fixed in next commit